gguf-py/gguf/gguf_writer.py

```diff
@@ -135,7 +135,7 @@ def get_total_parameter_count(self) -> tuple[int, int, int, int]:
 
                 size = prod(shape)
 
-                if "_exps." in name:
+                if "_exps." in name and len(shape) > 3:
```

Comment:
What's the rationale behind requiring stacked expert tensor to be at least 4-dimensional?

AFAIK, these are always 3-dimensional, and so this is equivalent to `if "_exps." in name and False:`, which doesn't look correct.

Reply:
Very good call! I had made this change while still trying to get things off the ground and forgot to rebate it out.

Reply:
I've rebased this out now